docs/deployment.md

```diff
@@ -1,24 +1,39 @@
 # Deployment
 
-All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md).
+All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md). During packaging, deployment binaries are created in the `deploy` directory, some of which are packaged into zip files in the `zips` folder, and other deployment artifacts are built from these pieces.
 
 ## ServiceControl installer
 
-A [windows installer is provided for download](https://docs.particular.net/servicecontrol/installation) containing both ServiceControl Management Utility (SCMU) and [Powershell commands](https://docs.particular.net/servicecontrol/powershell).
+The zips are packaged as embedded resources in the ServiceControl Management Utility (the "installer").
 
-When installing an instance using one of the above method instance files, persister (if applicable) and transport files are unzipped into the same folder.
+When installing an instance using one of the above method instance files, the following binaries are unzipped together to form a complete instance:
+
+- Application & persistence files
+- Selected transport
+- RavenDB server files (ServiceControl and Audit only)
 
 A configuration file generated based on:
 
 - User input (and defaults)
-- Contents of the `persister.manifest` file of the selected persister
-
-## NuGet package
+- Contents of the `transport.manifest` and `persister.manifest` file for the selected transport and persister.

```

Comment:
I know a version of this line already existed, but in what way are the contents of the manifest files used to create a config file?